nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -16,11 +14,13 @@ define([
             spec: app spec
     */
     function ConfigureWidget(options) {
-        const model = options.model,
-            spec = options.spec,
-            fileType = options.fileType,
-            runtime = Runtime.make();
-
+        const model = options.model, // the data model, inputs, params, etc.
+            spec = options.spec, // the Spec object
+            fileType = options.fileType, // which which filetype we're configuring here
```

Comment:
Given that the file type picker needs to be reworked so it only shows up when the configure tab is active (see [DATAUP-402 ](https://kbase-jira.atlassian.net/browse/DATAUP-402)), is there anything here that might need to be changed or thought about?

Reply:
Thought about, maybe. Fixed, probably not yet. Although, I guess we could put the file type picker into this widget and have it control that as well. But that's another PR.